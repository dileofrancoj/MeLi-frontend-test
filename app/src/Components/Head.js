import React from "react";
import Helmet from "react-helmet";

import PropTypes from "prop-types";
const Head = ({ title, description }) => {
  return (
    <Helmet>
      <title>{title}</title>
      <meta name="description" content={description} />
    </Helmet>
  );
};

Head.propTypes = {
  title: PropTypes.string,
  description: PropTypes.string,
};

export default Head;
