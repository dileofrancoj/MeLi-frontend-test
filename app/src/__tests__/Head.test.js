import React from "react";
import { shallow } from "enzyme";

import Head from "./../Components/Head";

describe("Header", () => {
  test("should render Head component with props ", () => {
    const component = shallow(
      <Head title="Title de prueba" description="Descripcion de prueba" />
    );
    expect(component).toMatchSnapshot();
  });
});
