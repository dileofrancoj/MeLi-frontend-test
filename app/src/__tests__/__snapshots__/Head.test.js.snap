// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Head component with props  1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <title>
    Title de prueba
  </title>
  <meta
    content="Descripcion de prueba"
    name="description"
  />
</HelmetWrapper>
`;
